javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $background = document.querySelector('body');
+var $lightBulb = document.querySelector('.light-bulb');
+var count = 0;
+$lightBulb.addEventListener('click', function (event) {
+  count++;
```

Comment:
Rather than using a counter variable then checking if the value is even or odd, I would suggest using a boolean to keep track of if the light is currently on or off, and re-assign the value to the opposite boolean every time the user clicks the light switch.